actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/javadsl/TestKitJunit5Extension.scala

```diff
@@ -23,7 +23,7 @@ final class TestKitJunit5Extension() extends AfterAllCallback with BeforeTestExe
    */
   override def beforeTestExecution(context: ExtensionContext): Unit = {
 
-    context.getTestInstance.ifPresent(instance => {
+    context.getTestInstance.ifPresent((instance: AnyRef)  => {
       val fielValue = AnnotationSupport.findAnnotatedFieldValues(instance, classOf[Junit5TestKit]).asScala.toList.head
```

Comment:
could you fix the val name? - should be `fieldValue`

also calling .head on a list is dangerous - could you check that there is actually something in the list first?

Reply:
fixed it

Reply:
you don't appear to have to fixed the issue with the unguarded call to `.head`

Reply:
actually, I see the isEmpty check in https://github.com/apache/incubator-pekko/pull/445/files#diff-a3b630cebfd1e64c3ee9e76b844d1e05a68e8f0e4712ba3e502d1088e8daec1dR28